src/database/migrations/0001_create_canonical_fields.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE canonical_fields (
+  id INTEGER PRIMARY KEY GENERATED ALWAYS AS IDENTITY,
+  label TEXT,
+  short_code TEXT,
+  field_type TEXT,
+  created_at TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT NOW()
+);
+
+comment on table canonical_fields is
+  'Canonical fields are those fields that are unique across organizations.';
+comment on column canonical_fields.label is 'The description of the field';
+comment on column canonical_fields.short_code is 'The short name of the field';
```

Comment:
Actually, this applies to all of these column comments: should we just use the more intuitive names for the field names?

e.g. `label` becomes `description and `short_code` becomes `short_name`

Reply:
Perhaps, yes. I think the main thing to document is what purpose the `short_code` serves as distinct from the `label` or `description`. Is there a "no spaces allowed in short_code" rule? Is the short_code going to be unique within an organization? Is it unique across organizations? Is it limited to 15 characters (and if so, why not char(15) or varchar(15))?